server/backend/users/views.py

```
@@ -130,6 +127,60 @@ def post(self, request):
+                fail_silently=False,
+            )
+
+            return Response({"message": "If this email exists, a password reset link has been sent."})
+        
+        except User.DoesNotExist:
+            # Do not reveal user existence for security
+            return Response({"message": "If this email exists, a password reset link has been sent."})
+        
+
+class PasswordResetConfirmView(APIView):
+    permission_classes = [AllowAny]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)